tests/autoyast/clone.pm

```diff
@@ -33,6 +33,11 @@ sub run {
     save_screenshot;
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    $self->export_logs;
+}
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). Why not defining a subclass of opensusebasetest for all autoyast modules? That should be really simple. There you can then have this post_fail_hook. If you see a specific reason why you did not want to have it, sure, LGTM :)

Reply:
Sorry, I tried wth y2logsstep, but it doesn't work well if we are on root console already, so found better base class which already does the trick.

Reply:
uhm, the "consoletest" base class has a post_run_hook executing "cd" after every module. That is certainly not what we want here. Even if that wouldn't be there the name would not match at all.

Reply:
What's about console_yasttest? I guess that one does the trick =)